src/Features/TestUtilities/EditAndContinue/Extensions.cs

```diff
@@ -109,6 +115,4 @@ public static ProjectInfo CreateProjectInfo(string projectName, string language
                     assemblyPath: Path.Combine(TempRoot.Root, projectName + ".dll"),
                     generatedFilesOutputDirectory: null))
                 .WithTelemetryId(CreateProjectTelemetryId(projectName));
-
-    }
```

Comment:
It looks like this file doesn't have a file-scoped namespace in the `release/dev17.14` branch.